ug: offline-update: Adjustments in return codes documentation (#711)
foundriesio/docs: source/user-guide/offline-update/offline-update.rst

    - Reboot is required to complete installation. After reboot ``aklite-offline run`` must be invoked.
- *101*: Installation succeeded.
    - Restart of dockerd service is required to complete installation, e.g. ``systemctl restart docker``. After the restart ``aklite-offline run`` must be invoked.
- *102*: Downgrade attempt.

>     - The Target to be installed has a version lower than the one that is running. The ``--force`` option can be used to force its installation.

Reviewer: ```suggestion
    - The Target to be installed has a version lower than the one running. Use the ``--force`` option to force its installation.
```

Author: Thanks, I've applied the suggested change.
